### ISSUE TITLE
MIDS Element - PhysicalSpecimenId

### ISSUE THREAD

**Issue by RBGE-Herbarium** [tdwg/mids#10]
**Summary**
MIDS information element: ModifiedOn (date/time of subsequent modification, if any)
DwC term (latest, 2014-11-08): modified
ABCD term name(3.0) : hasDateModified
Recommendations: Use [ISO 8601]

**Comment by dshorthouse** [comment]
We also have [`materialSampleID`](https://dwc.tdwg.org/terms/#dwc:materialSampleID). The example provided for a `MaterialSample` states:

> A whole organism preserved in a collection.

**Comment by matdillen** [comment]
A definition has been suggested: “whatever the institution uses to uniquely identify the item within that institute”

This could be a single identifier, a set of written directions or a combination of multiple properties. As indicated in #4, for meteorites the mass of the specimen is used, in addition to a name (MIDS-1), a date (MIDS-2) and/or a location (MIDS-2).

So, assuming all those properties are needed to uniquely identify a single meteorite in an institution, this would imply that PhysicalSpecimenId maps for that institution to the corresponding 4 properties in the data model it is employing (e.g. the local CMS model or ABCDEFG). Hence, the need for mass as a MIDS property seems absolved by the existence of PhysicalSpecimenId, as long as this MIDS-0 property can require data in multiple fields in the source dataset.

Similarly, an institution may map PhysicalSpecimenId strictly to `dwc:catalogNumber` or it may require an additional `dwc:collectionCode`, `dwc:kingdom` or `dwc:country` to uniquely identify the specimen.

The big question is how we will manage all those mappings and how we can harmonize them as much as possible. In this case, improving Darwin Core will not necessarily help because the problem is how individual institutions manage their specimens differently.

**Comment by tucotuco** [comment]
> We also have [`materialSampleID`](https://dwc.tdwg.org/terms/#dwc:materialSampleID). The example provided for a `MaterialSample` states:
> 
> > A whole organism preserved in a collection.

I agree completely. There seems to be no need to create a new term for this.

**Comment by emhaston** [comment]
Purpose: To allow the curator/researcher to identify the physical specimen the data refer to. To allow citation of the specimen. To enable the attachment of additional information to the specimen record.

**Comment by emhaston** [comment]
Element confirmed at TDWG Task Group meeting (5 May 2022)

For final version:

- Check need for constraints
- Name discussed again (Material / Specimen / …?)
- Check recommendations - many institutes use a barcode with alphanumeric codes

**Comment by emhaston** [comment]
Dublin core: identifier: http://purl.org/dc/elements/1.1/identifier

**Comment by emhaston** [comment]
Updated element metadata table to follow DWC format more closely
Deleted the Repeatable item | **Repeatable** | Yes. An object can have several identifiers.  |

**Comment by emhaston** [comment]
Include occurrenceID in the mapping.
Potentially recommend mapping physicalspecimenID to materialSampleID with the understanding that occurrenceID has a higher level mapping similar to CollectionObjectID in Specify.

**Comment by emhaston** [comment]
If materialSampleID maps exactly to the PhysicalSpecimenID then should the name of this element be changed? Also checking the LtC term of  ltc:RecordLevel.hasIdentifier 
https://github.com/tdwg/cd/issues/43 "The machine-actionable information profile for the collection description digital object."
https://github.com/tdwg/cd/issues/126 "A numeric, textual value, or reference such as an IRI, that can be used to uniquely identify the object to which it is attached."

**Comment by emhaston** [comment]
Maps narrowly to materialEntityID

**Comment by elspethhaston** [comment]
Accepted in specification for public review.
